### ISSUE TITLE
Unforking this repository

### ISSUE THREAD

**Issue by Danesprite** [dictation-toolbox/dragonfly#59]
There has recently been [discussion on Gitter](https://gitter.im/sphinx-dragonfly/Lobby?at=5c3a32de0999400604de2c59) on making GitHub code search work with this repository. At the time of writing, code search [doesn't work with forks](https://help.github.com/articles/searching-in-forks/). While it is not a must-have feature, it would be nice to have. Detaching this repository from t4ngo/dragonfly seems to be the best option for fixing this.

There are also other reasons for unforking. It looks like the project will also be easier to find via web searches, GitHub references (i.e. #XX) wouldn't inadvertently reference issues in the original project instead and pull requests would be easier to make from other branches in dictation-toolbox/dragonfly. I have asked GitHub support if there are any downsides to unforking.

Unless there is opposition to this change, we'll go ahead with it and mention that the project was forked from t4ngo/dragonfly in the readme and docs.

**Comment by LexiconCode** [comment]
I agree this would be the best course of action both from a long-term and short-term perspective.

**Comment by Danesprite** [comment]
I have given GitHub support the go-ahead now. Forks of this repository should remain intact. They have assured me there aren't any downsides other than not being able to submit pull requests to t4ngo/dragonfly.
